test/Sign.Cli.Test/Options/UriOptionTests.cs

```diff
@@ -10,15 +10,29 @@ public abstract class UriOptionTests : OptionTests<Uri?>
     {
         private static readonly Uri ExpectedValue = new("https://domain.test");
 
-        protected UriOptionTests(Option<Uri?> option, string shortOption, string longOption, bool isRequired)
-            : base(option, shortOption, longOption, ExpectedValue, isRequired)
+        protected UriOptionTests(Option<Uri?> option, string shortOption, string longOption)
+            : base(option, shortOption, longOption, ExpectedValue)
         {
         }
 
         [Fact]
         public void Option_WhenValueFailsToParse_HasError()
         {
-            VerifyHasError("3");
+            const string value = "3";
+
+            if (Option.IsRequired)
```

Comment:
Just curious how this `IsRequired` option gets `true`, `false` values? Who is passing it?

Reply:
`IsRequired` is set in the product and passed with the `Option` instance to the test.

https://github.com/dotnet/sign/blob/7b32935c42c720d96ceee799980a7f5ec2d2c296/src/Sign.Cli/CodeCommand.cs#L30-L32